lib/domain/provider/provider_model.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class ProviderModel<T> extends ChangeNotifier {
```

Comment:
wytłumaczysz do czego służy initialValue? jeżeli jest używane tylko raz, to może można całkiem się tego pozbyć i stosować samo value?

Reply:
Przy okazji, jeśli ma zostać to powinno się nazywać `fallbackValue` albo `defaultValue`.

Reply:
usunalem initialValue